### DISCUSSION TITLE
Supported inverters

### DISCUSSION THREAD

**Discussion by josh-sanders** [josh-sanders/home-assistant-omnik-trannergy-pv-inverter#3] [General]
Please comment on your experience with this custom component.

SUCCESS
Inverter: Trannergy SGN5400TL that has a wifi kit
Serial number starting with: 645xxxxxx
Home Assistant core version: 2024.10.3
home-assistant-omnik-trannergy-pv-inverter release: v0.2.1

**Comment by rob-on-git** [comment]
Inverter: Trannergy SGN4000TL wired
Serial beginning with PVL4000N...
Homeassistant core version: 2025.9.4
home-assistant-omnik-trannergy-pv-inverter release: v0.2.1

Works great already for quit a long time!
Thanks
